src/net/connection/quality.rs

```diff
@@ -25,17 +21,12 @@ impl NetworkQualityMeasurer {
 
     /// This will calculate the round trip time (rtt) from the given acknowledgement.
     /// Where after it updates the rtt from the given connection.
-    pub fn update_connection_rtt(
+    pub fn get_rtt(
```

Comment:
Okay, I see now from my earlier comment, why does a function with the name `get` not return anything?

Reply:
https://github.com/amethyst/laminar/blob/406126a74db681eef43b5d436eb007f70185f4dc/src/net/connection/quality.rs#L27